src/cmr/concepts/aclConcept.js

```diff
@@ -0,0 +1,344 @@
+    const { data } = jsonResponse
+    const { items } = data
+
+    return items
+  }
+
+  /**
+   * Rename fields, add fields, modify fields, etc
+   * @param {Object} item The item returned from the CMR json endpoint
+   */
+  normalizeJsonItem(item) {
+    return item
+  }
+
+  /**
+   * Set the search after identifier for a search against against the json endpoint
+   * @param {Float} searchAfterIdentifier The search after identifier provided by CMR
+   */
+  setJsonSearchAfter(searchAfterIdentifier) {
+    this.jsonSearchAfterIdentifier = searchAfterIdentifier
+  }
+
+  /**
+   * Retrieve the request id header from the request
+   * @param {Object} headers The provided headers from the query
+   * @return {String} Request ID defined in the headers
+   */
+  getRequestId() {
+    const {
+      'CMR-Request-Id': requestId
+    } = this.headers
+
+    return requestId
+  }
+
+  /**
+   * Retrieve the client id header from the request
+   * @param {Object} headers The provided headers from the query
+   * @return {String} Client ID defined in the headers
+   */
+  getClientId() {
+    const {
+      'Client-Id': clientId
+    } = this.headers
+
+    return clientId
+  }
+
+  /**
+   * Get the CMR concept type of this object
+   */
+  getConceptType() {
+    return this.conceptType
+  }
+
+  /**
+   * Returns an array of keys that should not be indexed when sent to CMR
+   */
+  getNonIndexedKeys() {
+    return [
+      'concept_id',
+      'offset',
+      'page_size',
+      'sort_key',
+      'permitted_user',
+      'include_full_acl',
+      'page_num',
+      'target'
+    ]
+  }
+
+  /**
+   * Retrieve a single item from the result set
+   * @param {String} id The concept id of an item to lookup in the result set
+   */
+  getItem(id) {
+    // If an item with the provided id does not exist, create one
+    if (Object.keys(this.items).indexOf(id) === -1) {
+      this.items[id] = {}
+    }
+
+    // Fetch the item from the result set
+    const { [id]: item } = this.items
+
+    return item
+  }
+
+  /**
+   * Set a specific property on an object within the result set of a search operation
+   * @param {String} id Concept ID to set a value for within the result set
+   * @param {String} key The key to set within the result
+   * @param {Any} value They value to assign to the key within the result
+   */
+  setItemValue(id, key, value) {
+    // Ensure that our object knows about the item by creating it if it doesn't exist
+    this.getItem(id)
+
+    // Set the value
+    this.items[id][key] = value
+  }
+
+  /**
+   * Parses the response from the json endpoint
+   * @param {Object} jsonResponse HTTP response from the CMR endpoint
+   * @param {Array} jsonKeys Array of the keys requested in the query
+   */
+  async parseJson(jsonResponse, jsonKeys) {
+    const { headers } = jsonResponse
+    const {
+      'cmr-hits': cmrHits,
+      'cmr-search-after': jsonSearchAfterIdentifier
+    } = downcaseKeys(headers)
```

Comment:
this use of douncaseKeys() looks good, don't know if other functions need to do this.

Reply:
This is also reused from the Concept component